style.css

```diff
@@ -107,3 +111,377 @@ body {
 .fab {
   color: #505f79;
 }
+
+/* works section starts here */
+
+.works-section-container {
+  background-color: #e5e5e5;
+  width: 100%;
+
+  /* border: 5px solid #df5819; */
```

Comment:
- [ ] [OPTIONAL] Kindly remove all commented code, it makes your code look messy